### PR TITLE
Made sure that HaplotypeCaller handles stubs correctly -- this completes issue #5079

### PR DESCRIPTION
**Pull request by MartonKN**

### CODE REVIEW

**Comment by davidbenjamin** [comment]
Closes #5079.

File: src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegionEvaluator.java [hunk 1]
```diff
@@ -19,4 +19,9 @@
      * @return ActivityProfileState containing the probability between 0.0 and 1.0 that the site is active
      */
     ActivityProfileState isActive( final AlignmentContext locusPileup, final ReferenceContext referenceContext, final FeatureContext featureContext );
+
+    // After trimming to fit the assembly window, throw away read stubs shorter than this length
+    // if we don't, the several bases left of reads that end just within the assembly window can
+    // get realigned incorrectly.  See https://github.com/broadinstitute/gatk/issues/5060
+    int MINIMUM_READ_LENGTH_AFTER_TRIMMING = 10;
```
[the remark this answers is not in the document]
Comment (MartonKN) [reply src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegionEvaluator.java]: 👍 Makes sense -- I replaced it

**Comment by davidbenjamin** [comment]
:thumbsup: ready to merge once the test passes (probably a transient Travis error)

**Comment by davidbenjamin** [comment]
Actually, the errors are not transient.  It's probably a matter of fixing the truth data.  But anyway, this still has my :thumbsup: once you resolve the failures.

**Comment by MartonKN** [comment]
Resolved in #5489

**Comment by davidbenjamin** [comment]
> Resolved in #5489

In that case, don't forget to close this version of the PR and delete the branch.

**Comment by MartonKN** [comment]
👍

**Pull request closed without merging**